gapis/api/vulkan/find_issues.go

```diff
@@ -44,17 +42,20 @@ const (
 // closed.
 // NOTE: right now this transform is just used to close chans passed in requests.
 type findIssues struct {
-	replay.EndOfReplay
+	replay.EndOfReplay2
 	state           *api.GlobalState
 	issues          []replay.Issue
 	reportCallbacks map[VkInstance]VkDebugReportCallbackEXT
+	allocations     *allocationTracker
 }
 
-func newFindIssues(ctx context.Context, c *capture.GraphicsCapture) *findIssues {
+func NewFindIssues(ctx context.Context, c *capture.GraphicsCapture) *findIssues {
```

Comment:
Who is using this outside of the vulkan package. If no-one it should be private.